.travis.yml

```diff
@@ -36,6 +36,7 @@ env:
 
 install:
   - "pip install -r requirements.txt"
+  - "pip install -r requirements/test-local.txt"
```

Comment:
Yep, the plugin tries to import `settings` from `django.conf` which fails without the app installed [1], so we need this.

[1] http://webqa-ci-staging1.qa.scl3.mozilla.com:8080/job/oneanddone.adhoc/55/console